docs/introduction/index.md

```diff
@@ -218,7 +218,21 @@ This exposes [`CommandService`](concepts.html#command-service),
 
 ## Repeating the cycle
 
+The stages described above are repeated as another Bounded Context is added to the implementation,
+or as changes or extensions to the existing contexts are required. 
+ 
 ## Client application development
+
+Development of client applications may start after the [data types are defined](#defining-data-types).
+and Spine Model Compiler generates the code for all supported client platforms.
```

Comment:
Something is not right here. There is a period right before the start of this line.